### DISCUSSION TITLE
Make workflows use DXO only

### DISCUSSION THREAD

**Discussion by holgerroth** [NVIDIA/NVFlare#1084] [Ideas]
Right now, looking at the workflows, it is somewhat confusing as to what goes into `Shareable` and what goes into `DXO`. 
We might be able to reduce that potential for confusion by utilizing only `DXO` in workflows and executors. The workflow/executor developer shouldn't need to worry about additional information that is put into `Shareable` by the framework.

**Comment by holgerroth** [comment]
@yanchengnv , @chesterxgchen, @ZiyueXu77 , @Can-Zhao

**Reply by yanchengnv** [reply]
Agreed. We can refactor existing controllers and executors fairly easily.
